src/WebJobs.Script.WebHost/WebJobs.Script.WebHost.csproj

```diff
@@ -57,7 +57,7 @@
     <PackageReference Include="Microsoft.Azure.WebJobs.Host.Storage" Version="3.0.11" />
     <PackageReference Include="Microsoft.Azure.WebJobs.Logging" Version="3.0.11" />
 	  <PackageReference Include="Microsoft.Azure.KeyVault" Version="3.0.3" />
-    <PackageReference Include="Microsoft.Azure.WebSites.DataProtection" Version="2.1.88-alpha" />
+    <PackageReference Include="Microsoft.Azure.WebSites.DataProtection" Version="2.1.999-alpha" />
```

Comment:
This version is from staging feed. Once https://github.com/Azure/azure-websites-security/pull/9 is merged this should be updated with the new version from production feed.